### PR TITLE
Updated CSR configuration

### PR DESCRIPTION
**Pull request by KDNeufeld**
Added docker commands to run openssl to regenerate/verify the csr.
Bumped the bits from 2048 to 4096

### CODE REVIEW

File: infrastructure/certificates/tickets.gov.bc.ca.conf [hunk 1]
```diff
@@ -3,9 +3,13 @@
 #
 # Generate CSR
 #   openssl req -new -out tickets.gov.bc.ca.csr -newkey rsa:2048 -nodes -sha256 -keyout tickets.gov.bc.ca.key -config tickets.gov.bc.ca.conf
+#   or using docker
+#   docker run -it --rm -v %cd%:/apps -w /apps alpine/openssl req -new -out tickets.gov.bc.ca.csr -newkey rsa:2048 -nodes -sha256 -keyout tickets.gov.bc.ca.key -config tickets.gov.bc.ca.conf
 #
 # Verify CSR
 #   openssl req -test -noout -verify -in tickets.gov.bc.ca.csr
```
Comment (KDNeufeld) [inline infrastructure/certificates/tickets.gov.bc.ca.conf:10]: Note: -test gives me an error - I think this command is invalid.  Using "-text" does seem to work, however.
```
req: Unrecognized flag test
req: Use -help for summary.
```

Comment (KDNeufeld) [reply infrastructure/certificates/tickets.gov.bc.ca.conf]: Fixed the -test typo.

**Review by pbolduc — APPROVED** [review]

File: infrastructure/certificates/tickets.gov.bc.ca.conf [hunk 2]
```diff
@@ -3,15 +3,19 @@
 #
 # Generate CSR
 #   openssl req -new -out tickets.gov.bc.ca.csr -newkey rsa:2048 -nodes -sha256 -keyout tickets.gov.bc.ca.key -config tickets.gov.bc.ca.conf
```
Comment (pbolduc) [inline infrastructure/certificates/tickets.gov.bc.ca.conf:5]: could you update this rsa:2048 too

Comment (KDNeufeld) [reply infrastructure/certificates/tickets.gov.bc.ca.conf]: done

**Review by pbolduc — APPROVED** [review]

**Pull request merged**